### DISCUSSION TITLE
Is it possible to use Whisper to Translate&Transcribe for a different language than English? English speech to Spanish transcribe

### DISCUSSION THREAD

**Discussion by zivafgin** [openai/whisper#1715] [Q&A]
Hi everyone,
Is it possible to use Whisper to Translate&Transcribe for a different language than English? English speech to Spanish transcribe

Thanks in advance,
Ziv

**Comment by AIhasArrived** [comment]
Did you find an answer?

**Reply by zivafgin** [reply]
nope :(

**Comment by phineas-pta** [comment]
no, use other translation library/service
